docs/application/native/guides/compliations/watchface-complication.md

```diff
@@ -25,25 +25,31 @@ Watch Face can set the complication support types to support specific type of co
 
 The followings are types of watch face complication:
 
-| Support type name | Mandatory data | Optional data   |
-|-------------------|----------------|-----------------|
-| short text type   | short text     | title           |
-|                   |                | icon path       |
-|                   |                | extra data      |
-| long text type    | long text      | title           |
-|                   |                | icon path       |
-|                   |                | extra data      |
-| ranged value type | min value      | title           |
-|                   | max value      | short text      |
-|                   | current value  | icon path       |
-|                   |                | extra data      |
-| image type        | image path     | extra data      |
-| icon type         | icon path      | extra data      |
-| time type         | timestamp      | short text      |
-|                   |                | extra data      |
-| empty             |                |                 |
-
- 
+| Support type name | Mandatory data   | Optional data   |
+|-------------------|------------------|-----------------|
+| short text type   | short text       | title           |
+|                   |                  | icon path       |
+|                   |                  | extra data      |
+| long text type    | long text        | title           |
+|                   |                  | icon path       |
+|                   |                  | extra data      |
+| ranged value type | min value        | title           |
+|                   | max value        | short text      |
+|                   | current value    | icon path       |
+|                   |                  | extra data      |
+| image type        | image path       | extra data      |
+| icon type         | icon path        | extra data      |
+| time type         | timestamp        | short text      |
+|                   | time information | extra data      |
+| empty             |                  |                 |
+
+> **Note**

```

Comment:
Please remove \<br/> and use another \> to make a line break.
Also, please use same indentation before the \>s

```
> **Note**
>
> `timestamp` data is deprecated since Tizen 5.5.
> Instead, use `time information` data.
> The time type should provide `timestamp` or `time information` but not both.
```